runtime/interface.go

```diff
@@ -60,59 +62,67 @@ type Interface interface {
 	DecodeArgument(b []byte, t cadence.Type) (cadence.Value, error)
 }
 
+type Metrics interface {
+	ProgramParsed(duration time.Duration)
+	ProgramChecked(duration time.Duration)
+	ProgramInterpreted(duration time.Duration)
+	ValueEncoded(duration time.Duration)
+	ValueDecoded(duration time.Duration)
+}
```

Comment:
Could we make this interface report the start and end timestamps rather than the elapsed time? This would allow us to construct traces from these metrics and place them on the larger execution timeline:

![Screen Shot 2020-05-05 at 10 43 26 AM](https://user-images.githubusercontent.com/2547035/81097852-512ba780-8ebd-11ea-908a-15af19dc4a54.png)

Reply:
Great idea! I'll change this quickly

Reply:
Changed in 05c813c8847772df6eb85fa75dae4753c331f74e